src/main/java/de/unistuttgart/crosswordbackend/crosswordchecker/CrosswordGenerator.java

```diff
@@ -0,0 +1,261 @@
+package de.unistuttgart.crosswordbackend.crosswordchecker;
+
+import de.unistuttgart.crosswordbackend.data.Crossword;
+import de.unistuttgart.crosswordbackend.data.Intersection;
+import de.unistuttgart.crosswordbackend.data.Question;
+import java.security.SecureRandom;
+import java.util.*;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class CrosswordGenerator {
+
+  /**
+   * Count of crosswords generated and compared by getScore() function
+   */
+  private int interations = 10;
+  private int startRows = 10;
+  private int startColumns = 10;
+
+  private final String empty = "empty";
+
+  private SecureRandom random = new SecureRandom();
+
+  /**
+   * Count of tries of the algorithm to place a word.
+   */
+  private int maxTries = 100;
+
+  /**
+   * Genereates a crossword with the given questions
+   * @param questions Questions to create a crossword-puzzle with
+   * @return a Crossword-puzzle
+   */
+  public Crossword generateCrossword(List<Question> questions) {
+    Crossword crossword = createCrossword(questions);
+    for (int i = 0; i < interations; i++) {
+      Crossword currentCrossword = createCrossword(questions);
+      int crosswordScore = getScore(crossword);
+      int currentCrosswordScore = getScore(currentCrossword);
+      if (currentCrosswordScore > crosswordScore && currentCrossword.getAnswer().isEmpty()) {
+        crossword = currentCrossword;
+      }
+    }
+
+    return crossword;
+  }
+
+  /**
+   * Converts the Array from type question to an Array with the type
+   * String. The String Array only contains the answers.
+   * Then the simpleCrossword algorithm is called.
+   * @param questions Question Array who will be converted
+   * @return Crossword out of the given questions
+   */
+  private Crossword createCrossword(List<Question> questions) {
+    ArrayList<String> answers = new ArrayList<>();
+    questions.forEach(question -> answers.add(question.getAnswer().toUpperCase()));
+    Crossword crossword = simpleCrossword(answers);
+    crossword.setQuestions((ArrayList<Question>) questions);
+    crossword.setAnswer(answers);
+    return crossword;
+  }
+
+  /**
+   * Simple algorithm generate a crossword out of given answers.
+   * @param answers List of answers which will be placed in a crossword if possible.
+   * @return crossword with the given answers.
+   */
+  private Crossword simpleCrossword(ArrayList<String> answers) {
```

Comment:
```suggestion
  private Crossword simpleCrossword(List<String> answers) {
```

Reply:
Also, that method should probably be renamed to something like `generateCrossword`...